device/config.py

```diff
@@ -110,6 +110,8 @@ def load(self, toml_path):
         self.cleardarksky: bool = self.get_toml('webui_settings', 'cleardarksky', False)
         self.experimental: bool = self.get_toml('webui_settings', 'experimental', False)
         self.confirm: bool = self.get_toml('webui_settings', 'confirm', True)
+        self.save_frames: bool = self.get_toml('webui_settings', 'save_frames', False)
+        self.save_frames_dir: str = self.get_toml('webui_settings', 'save_frames_dir', '.')
```

Comment:
These need rendering in render_config_html - and parsing in load_from_form (unless intended to only be in the toml file, and not modifiable in the UI)